models/user.go

```diff
@@ -1,7 +1,38 @@
 package models
 
+import (
+	"time"
+
+	"github.com/google/uuid"
+	"gorm.io/gorm"
+)
+
 type Users struct {
```

Comment:
noted 
add all models
createdBy
updatedAt
updatedby

ATAU SAYA PECAT ANDA YAA
WKAOKWOAKWOKAOWKOAKW

Reply:
![Screenshot (45)](https://github.com/Saadyyyy/Email-Verification/assets/66883583/e3508c3a-b0a4-495c-8074-b0e49aed4138)

- untuk model user i think you need new model, toh itu by siapa ? klo ada model admin ya its ok
- updatedAt & updatedby di model token, mau di terapkan? karena itu cuman temporary data aja cuman butuh kapa token dibuat

Reply:
ANDA DI PECAT

Reply:
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce efficitur velit ac nunc feugiat aliquam. Quisque pretium euismod tellus nec dictum. Cras elementum urna sit amet pulvinar sollicitudin. Ut rhoncus ante sed tempus euismod. Integer vitae leo ac leo tristique facilisis. Proin efficitur elit urna, eu semper tortor sagittis vitae. Maecenas eget felis nunc. Aliquam sollicitudin tempus nulla ut venenatis. Mauris id diam vitae augue rhoncus blandit pellentesque ac ipsum. Aenean dui libero, volutpat quis dapibus eu, suscipit quis ex. Etiam eu dui sit amet enim euismod sollicitudin.

Aliquam quis lobortis nibh, nec volutpat risus. Praesent eget lacus dignissim, gravida quam vitae, porttitor ligula. Sed malesuada turpis at blandit lobortis. Pellentesque ut faucibus odio. Cras vulputate fermentum tempor. Curabitur ante neque, ornare ut efficitur vitae, sollicitudin ut nibh. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse venenatis dolor elit, eu lobortis mauris cursus ac. Vestibulum a viverra dui. Ut auctor eros lectus, sit amet pharetra erat vestibulum at. Vivamus cursus nec ante nec scelerisque.